package.json

```diff
@@ -4,6 +4,7 @@
   "description": "M3U channels list splitter and serves channel links",
   "main": "application.js",
   "scripts": {
+    "start": "node application.js",
```

Comment:
it is recommended adding "$@" at the end of execution script ;)

Reply:
Good point. I hope you know what is it used for... :rofl: